inst/scribble/scribble.js

```diff
@@ -0,0 +1,626 @@
+/*
+ *  Scribble for remark.js/xaringan
+ *  Matthew T. Warkentin <warkentin@lunenfeld.ca>
+ *  and Garrick Aden-Buie <garrick@adenbuie.com>
+ */
+
+/* global slideshow,fabric */
+
+class Scribble {
+  constructor (opts) {
+    // User options
+    this.opts = opts
+    this.penColors = opts.pen_color || '#000000'
+    this.currColor = this.penColors[0]
+    this.penSize = opts.pen_size || 3
+    this.eraserSize = opts.eraser_size || 30
```

Comment:
Same as above, this fallback could be updated to `this.penSize * 10` to match `R` default.